src/components/TextInput/TextInput.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../theme/fonts.scss';
+@import '../../theme/theme.scss';
+
+:root {
+  --tl-text-focused: var(--color-gs00);
+}
+
+.kai-text-input {
+  padding: 1rem;
+
+  &-label {
+    padding-bottom: 0.5rem;
+    display: block;
+    color: var(--color-epsilon);
+    font-size: 1.2rem;
+  }
+
+  &-input {
+    display: block;
+    width: 100%;
+    box-sizing: border-box;
+    line-height: 3.6rem;
+    padding: 0 0.5rem;
+    font-size: 1.6rem;
```

Comment:
Don't hard code font sizes, please create a variable in `fonts.scss` so we have a single place we can reconfigure font sizes

Reply:
Fair point, will do! I quite often need to be called out for doing things like this. 😅 I'll try keep it to a minimum in future. 😶